Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/ProjectionIntegrityViolationDetector.php

```diff
@@ -182,20 +182,26 @@ public function subtreeTagsAreInherited(): Result
         // NOTE:
         // This part determines if a parent hierarchy relation contains subtree tags that are not existing in the child relation.
         // This could probably be solved with JSON_ARRAY_INTERSECT(JSON_KEYS(ph.subtreetags), JSON_KEYS(h.subtreetags) but unfortunately that's only available with MariaDB 11.2+ according to https://mariadb.com/kb/en/json_array_intersect/
-        $hierarchyRelationsWithMissingSubtreeTags = $this->client->getConnection()->executeQuery(
-            'SELECT
-              ph.name
-            FROM
-              ' . $this->tableNamePrefix . '_hierarchyrelation h
-              INNER JOIN ' . $this->tableNamePrefix . '_hierarchyrelation ph
-                ON ph.childnodeanchor = h.parentnodeanchor
-                AND ph.contentstreamid = h.contentstreamid
-                AND ph.dimensionspacepointhash = h.dimensionspacepointhash
-            WHERE
-              EXISTS (
-                SELECT t.tag FROM JSON_TABLE(JSON_KEYS(ph.subtreetags), \'$[*]\' COLUMNS(tag VARCHAR(30) PATH \'$\')) t WHERE NOT JSON_EXISTS(h.subtreetags, CONCAT(\'$.\', t.tag))
-              )'
-        )->fetchAllAssociative();
+        $query = 'SELECT
+          ph.name
+        FROM
+          ' . $this->tableNamePrefix . '_hierarchyrelation h
+          INNER JOIN ' . $this->tableNamePrefix . '_hierarchyrelation ph
+            ON ph.childnodeanchor = h.parentnodeanchor
+            AND ph.contentstreamid = h.contentstreamid
+            AND ph.dimensionspacepointhash = h.dimensionspacepointhash
+        WHERE
+          EXISTS (
+            SELECT t.tag FROM JSON_TABLE(JSON_KEYS(ph.subtreetags), \'$[*]\' COLUMNS(tag VARCHAR(30) PATH \'$\')) t WHERE NOT JSON_EXISTS(h.subtreetags, CONCAT(\'$.\', t.tag))
+          )';
```

Comment:
nitpick: It makes sense to put this into a variable, but the indentation is wrong, I would suggest to use Nowdoc like
$query = <<<'SQL'
...
SQL;